### PR TITLE
SW-4881 Add participant dropdown to internal deliverables table

### PR DESCRIPTION
**Pull request by nickgraz**
- Implement participant list redux async thunk
- Add `ParticipantDropdown` component
- Add dropdown to accelerator admin deliverables table

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* **#2205** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2205?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2205?utm_source=stack-comment).

**Review by joshbuchea — APPROVED** [review]
Nice and clean 👌

File: src/hooks/useParticipants.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
```
Comment (karthikbtf) [inline src/hooks/useParticipants.ts:1]: I messed up earlier moving useProjects to src/hooks. Keeping this with the component context works just fine. Thoughts?

File: src/redux/features/participants/participantsSlice.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { Participant } from 'src/types/Participant';
+import { buildReducers, StatusT } from 'src/redux/features/asyncUtils';
+import { requestParticipantsList } from 'src/redux/features/participants/participantsAsyncThunks';
+
+/**
+ * Participant list
+ */
+const initialStateParticipantsList: { [key: string]: StatusT<Participant[]> } = {};
```
Comment (karthikbtf) [inline src/redux/features/participants/participantsSlice.ts:9]: `Record<string, StatusT<Participant[]>>` ?

File: src/types/Participant.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export type Participant = {
```
Comment (karthikbtf) [inline src/types/Participant.ts:1]: should we group this under the parent Deliverables.ts?

**Review by karthikbtf — APPROVED** [review]
LGTM

**Comment by nickgraz** [comment]
### Merge activity

* **Feb 27, 11:22 AM EST**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2205).

**Pull request merged**